tests/integration_tests/test_chainermn.py

```diff
@@ -222,13 +222,8 @@ def objective(_trial: ChainerMNTrial, _comm: bool) -> float:
     @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
     def test_relative_sampling(storage_mode: str, comm: CommunicatorBase) -> None:
 
-        relative_search_space = {
-            "x": distributions.FloatDistribution(low=-10, high=10),
-            "y": distributions.FloatDistribution(low=20, high=30, log=True),
-            "z": distributions.CategoricalDistribution(choices=(-1.0, 1.0)),
-        }
-        relative_params = {"x": 1.0, "y": 25.0, "z": -1.0}
-        sampler = DeterministicRelativeSampler(relative_search_space, relative_params)
+        params = {"x": 1.0, "y": 25.0, "z": -1.0}
+        sampler = DeterministicSampler(params)
```

Comment:
I'm wondering if this `test_relative_sampling` is any meaningful at all. Do you think we should keep it? I'm neutral but the test is a bit confusing with "relative" in its name while using an "independent sampler".

Reference https://github.com/optuna/optuna/pull/380

Reply:
Thank you for your comment!
As you said, it looks that this test is not meaningful at all.

I don't think we should keep this test case since the test case is still passed although I changed to independent_sampler. I pushed https://github.com/optuna/optuna/pull/3827/commits/361874d5d0ba2a2ec198d3a59540c22ecb47c0d7.

PTAL.

Reply:
Sounds good, thanks for the quick fix.